Server/mods/deathmatch/logic/CPickup.cpp

```diff
@@ -61,6 +61,23 @@ CPickup::~CPickup ( void )
 }
 
 
+CElement* CPickup::Clone ( bool* bAddEntity, CResource* pResource )
+{
+    CPickup* pTemp = m_pPickupManager->Create ( GetParentEntity () );
+    if ( pTemp )
+    {
+        pTemp->SetPickupType ( GetPickupType () );
+        pTemp->SetPickupType ( GetPickupType () );

```

Comment:
Don't think setting the pickup type twice is required is it ?

Reply:
Removed, thx